CHANGELOG.rst

```diff
@@ -4,6 +4,10 @@ Vultr Collection Release Notes
 
 .. contents:: Topics
 
+v1.1.0
+======
+
+- vultr_block_storage - Included ability to resize, attach and detach volumes.
```

Comment:
this is a generated file do not touch. 

please add a changelog fragment to _changelogs/fragments_

e.g. 
```yaml
minor_changes:
  - vultr_block_storage - Included ability to resize, attach and detach volumes.
```
also see https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment

Reply:
Thanks. I was not familiar with this at all.

I've created the fragment. Please make sure it makes sense to you